stacks-signer/src/signerdb.rs

```diff
@@ -858,9 +858,13 @@ impl SignerDb {
         &self,
```

Comment:
I think a couple things should be done to make this more maintainable:

* This function should return an unsigned integer big enough to represent the maximum number of blocks in a tenure (a `u64` should be fine). There is no possible way there could be a negative number of blocks.

* If the tenure is not found, then an error (`DBError::NotFoundError`) should be returned.